src/feedback/feedback.tsx

```diff
@@ -0,0 +1,39 @@
+import {AccessibilityInfo} from 'react-native';
+export {voiceFeedback};
+
+let feedbackStr = new Map([
```

Comment:
Don't use a map, just declare an object/dictionary. Or if you prefer, declare typescript types for this.